src/app/models/gtypes/album_model.rs

```diff
@@ -85,6 +98,7 @@ mod imp {
             Self {
                 album: RefCell::new(None),
                 artist: RefCell::new(None),
+                year: RefCell::new(0),
```

Comment:
I feel like this could definitely be a RefCell<Option<u32>>, this would avoid all the unwrapping and awkward comparison to 0 you have to deal with!

Reply:
I was still confused how GTK deals with optional values. Fixed in the latest commit. :)